nightlies/cron_wrapper

```diff
@@ -37,6 +41,16 @@ if [ $code != 0 ] ; then
 else
         # normal exit: just log stdout
         DATE=$(date)
-        echo -n "$DATE: " >> $LOG
-        cat $STDOUT  >> $LOG
+        echo -n "$DATE: "  >> $LOG_XX
+        cat $STDOUT   >> $LOG_XX
```

Comment:
So, given that you still have STDOUT, and given that it's private to this process, there's really no need for LOG_XX; you could just echo the Running command... and the date directly to $STDOUT and remove LOG_XX altogether